thinkst/canarytokens
# Ensure we handle and redirect old links correctly

### Proposed changes
- This PR aims to rework the vue-router configuration to ensure we still handle links created on the older tokens.org
- The fix here is by removing the absolute base  pathname `/newuiwhodis/` to the base `'/'` and instead add the prefixes inside the router
- Also updates the navbar links correctly

### How to test
- Pull the branch locally
- Test that the navbar and footer links still work
- Create a token - it should still use the new `/manage` and `/history` links
- Try fetch the pages using `/manage/?auth=auth&token=token` and `/history/?auth=auth&token=token`. They should still work correctly
